packages/shared/styles/components/SfSection.scss

```diff
@@ -0,0 +1,14 @@
+@import '../variables';
+
+$section--spacing: 100px;
+$section--mobile-breakpoint: $mobile-max;
+
+.sf-section {
```

Comment:
@przspa don't you think the section may limit the width by default, such as a container?
Then it should also have `$section-max-width` SCSS var.

Reply:
and it would also have `--fluid` modifier to set full width

Reply:
@leomp12 I think it's better if it will adjust itself to a parent container